lib/chat/rooms.ex

```diff
@@ -158,7 +154,22 @@ defmodule Chat.Rooms do
   end
 
   defdelegate update(room), to: Registry
-  defdelegate decrypt_identity(encrypted_room_identity, person_identity, room_pub_key), to: Room
+
+  defdelegate link_message(room, room_identity, msg_id), to: RoomMessageLinks, as: :create
+  defdelegate message_link_hash(msg_id), to: RoomMessageLinks, as: :link_hash
+  defdelegate get_message_link(link_hash), to: RoomMessageLinks, as: :get
```

Comment:
Broker should know about room, room should know nothing about broker. Not in core context at least